core/include/cubos/core/al/audio_context.hpp

```diff
@@ -91,13 +95,7 @@ namespace cubos::core::al
         {
         public:
             virtual ~Buffer() = default;
-
-            /// @brief Fills the buffer with data.
-            /// @param format Audio format of the data.
-            /// @param size Size of the buffer in bytes.
-            /// @param data Buffer data.
-            /// @param frequency Audio frequency.
-            virtual void fill(Format format, std::size_t size, const void* data, std::size_t frequency) = 0;
+            virtual size_t getLength() = 0;
```

Comment:
```suggestion
            virtual size_t length() = 0;
```
Also missing docs, what does this return?